eHealthAfrica/direct-delivery
# implement login feature

#### What does this PR do?
Implements the login feature to multiple environments

#### Description of Task to be completed?
Created environment db on `[devURL]:5984/environment_db`
Replicates environment db to mobile app
Authenticate users via `[devURL]/ums/auth`
Retrieve user object with auth token from `[devURL]/ums/users/me`
Store user object on local `pouchdb`
Implement `react-redux` for environments and login

#### How should this be manually tested?
`react-native run-android`
Login with a driver account, you should be redirected to the main activity
Test with other accounts or wrong credentials

#### Any background context you want to provide?

#### What are the relevant JIRA stories?
[HDSLD-418](https://jira.ehealthafrica.org/browse/HDSLD-418)

#### Screenshots (if appropriate)
![screenshot_20171012-165623](https://user-images.githubusercontent.com/16849953/31506107-706458da-af6e-11e7-9895-a341cdc9f42d.png)

#### Questions:
